docs/guides/debug-qiskit-runtime-jobs.ipynb

```diff
@@ -0,0 +1,564 @@
+    "\n",
+    "# Use the same observables as before\n",
+    "pubs = [(qc, isa_obs)]\n",
+    "clifford_pubs = analyzer.to_clifford(pubs)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 48,
+   "id": "d63f55da-7744-46e2-a2f3-7362911e470a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "noise_factors = [\n",
+    "    [1, 1.1],\n",
+    "    [1, 1.1, 1.2],\n",
+    "    [1, 1.5, 2],\n",
+    "    [1, 1.5, 2, 2.5, 3],\n",
+    "    [1, 4],\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 49,
+   "id": "3fb22cc4-6c81-489c-beba-23c53a0c965e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Run the PUBs on a QPU\n",
+    "estimator = Estimator(backend)\n",
+    "estimator.options.default_shots = 100000\n",
+    "estimator.options.twirling.enable_gates = True\n",
+    "estimator.options.twirling.enable_measure = True\n",
+    "estimator.options.twirling.shots_per_randomization = 100\n",
+    "estimator.options.resilience.measure_mitigation = True\n",
+    "estimator.options.resilience.zne_mitigation = True\n",
+    "estimator.options.resilience.zne.amplifier = \"pea\"\n",
+    "\n",
+    "jobs = []\n",
+    "for factors in noise_factors:\n",
+    "    estimator.options.resilience.zne.noise_factors = factors\n",
+    "    jobs.append(estimator.run(clifford_pubs))\n",
+    "\n",
+    "results = [job.result() for job in jobs]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 50,
+   "id": "a326fd18-30cf-470b-af54-47ac5b113f78",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Perform a noiseless simulation\n",
+    "ideal_results = analyzer.ideal_sim(clifford_pubs)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 51,
+   "id": "a9344fbf-06ad-44c9-bd86-c3a50527d184",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Mean rel. diff. for factors [1, 1.1]:\n",
+      "  76.88%\n",
+      "\n",
+      "Mean rel. diff. for factors [1, 1.1, 1.2]:\n",
+      "  8.23%\n",
+      "\n",
+      "Mean rel. diff. for factors [1, 1.5, 2]:\n",
+      "  17.5%\n",
+      "\n",
+      "Mean rel. diff. for factors [1, 1.5, 2, 2.5, 3]:\n",
+      "  12.22%\n",
+      "\n",
+      "Mean rel. diff. for factors [1, 4]:\n",
+      "  12.73%\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# By looking at the mean relative difference we can quickly tell the best choice for our options\n",
+    "for factors, res in zip(noise_factors, results):\n",
+    "    d = rdiff(ideal_results[0], res[0])\n",
+    "    print(\n",
+    "        f\"Mean rel. diff. for factors {factors}:\\n  {np.round(np.mean(d), 2)}%\\n\"\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4542a9c8-273f-4fef-9954-6ca420dd5b4f",
+   "metadata": {},
+   "source": [
+    "The result with the smallest difference would help us decide which options to choose."
+   ]
```

Comment:
The page needs a Next steps section with recommendations for related guides and tutorials - what links would be appropriate here?